pkg/tracking/store/sql/experiments.go

```diff
@@ -78,6 +78,17 @@ func (s TrackingSQLStore) CreateExperiment(input *protos.CreateExperiment) (stri
 	return strconv.Itoa(int(*experiment.ID)), nil
 }
 
+func (s TrackingSQLStore) RenameExperiment(experiment *protos.Experiment) *contract.Error {
+	if err := s.db.Model(&models.Experiment{}).
```

Comment:
I don't think we are checking if the experiment life cycle is Active.
Python code does:
```
            if experiment.lifecycle_stage != LifecycleStage.ACTIVE:
                raise MlflowException("Cannot rename a non-active experiment.", INVALID_STATE)
```
We really want to do the same thing there.